indico/modules/rb/client/js/components/SidebarMenu.jsx

```diff
@@ -103,35 +107,106 @@ function SidebarMenu({
         }
     ].filter(({onlyIf}) => onlyIf === undefined || onlyIf);
 
+    const [contactVisible, setContactVisible] = useState(false);
+    const [termsVisible, setTermsVisible] = useState(false);
+    const [config, setConfig] = useState(null);
+
+    useEffect(() => {
```

Comment:
this would be a good case for `useIndicoAxios`, but see my comment above about this data already being available